readthedocs/rtd_tests/tests/test_config_integration.py

```diff
@@ -174,31 +203,67 @@ def test_extra_requirements(self, load_config):
         config = load_yaml_config(self.version)
         self.assertEqual(config.extra_requirements, [])
 
-    def test_conda(self, load_config):
-        to_find = '__init__.py'
-        load_config.side_effect = create_load({
-            'conda': {
-                'file': to_find
-            }
-        })
+    @mock.patch('readthedocs.projects.models.Project.checkout_path')
+    def test_conda_with_cofig(self, checkout_path):
+        base_path = tempfile.mkdtemp()
```

Comment:
I wasn't able to reuse the `apply_fs` function (only works with pytest) or `fake_paths` as the check is very strict.